definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -0,0 +1,100 @@
+[
```

Comment:
The story number in your PR's title is wrong. Can you please edit it?

Reply:
Im not sure what you mean. The ticket is 6713 and so is this story.
There may have been some confusion earlier, as i had PR-633, that was similar to this but closed hyperlinked under the PR-634 somehow. Was that what you were refering to ?

Reply:
Ignore me. After writing this I realised I could edit the PR's title (and so I did).
Forgot to delete this comment.